Resources/doc/index.rst

```diff
@@ -41,12 +41,11 @@ application:
 
     # config/packages/doctrine_migrations.yaml
 
-    doctrine_migrations:
         # List of namespace/path pairs to search for migrations, at least one required
         migrations_paths:
-            'App\Migrations': '%kernel.project_dir%/src/App'
-            'AnotherApp\Migrations': '/path/to/other/migrations'
-            'SomeBundle\Migrations': '@SomeBundle/Migrations'
+            # namespace is arbitrary but should be different from App\Migrations
+            # as migrations classes should NOT be autoloaded
```

Comment:
As I understand it, that migrations should not be autoloaded by the Symfony service container build process.

Reply:
Hmmm isn't what you're thinking of called auto discovery, or do you really mean PHP autoloading ?

Reply:
No, I didn't mean autoloading in the sense of "class autoloading", as provided by the composer autoloader. I meant being loaded and configured by the dependency injection component.

In any case, not using an "App\" namespace would likely prevent both things in most scenarios targeted by the docs, so not sure if the difference is of actual relevance.

But frankly, I just meant to duplicate what the recipe said, since the docs seemed to me to be deferring to the recipe; and only including a default config for cases where Flex wasn't installed, and the recipe wasn't applied.

I actually didn't really think through if the recipe was actually right and the docs wrong, or vice versa. Maybe it's better I close my PR and let someone more knowledgeable or with more time to research to help with this.

Sorry I can't be of more help.

Reply:
I was asking just in case you knew, or had the time to research, but I think I can find out. 

Following the last link you provided, a few extra clicks lead to https://github.com/symfony/recipes/pull/172

A bit more clicking leads to https://github.com/symfony/recipes/issues/343#issuecomment-721705155, which corroborates that it really is PHP autoloading that is meant here. I'm not saying I understand why autoloading a migration class would break the mechanism that loads migration classes from a directory, but I think PHP autoloading is really what was meant here. Maybe @stof can help with this?